src/utils/url.ts

```diff
@@ -0,0 +1,12 @@
+import { ChainNameOrId, MultiProvider } from '@hyperlane-xyz/sdk';
+import BigNumber from 'bignumber.js';
+
+export async function tryGetBlockExplorerAddressUrl(
+  multiProvider: MultiProvider,
+  chain: ChainNameOrId,
+  address: string,
+) {
+  return address && !new BigNumber(address).isZero()
```

Comment:
consider this helper from the utils package https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/583ea2525fde7843a66a6fe6069017ee737c4a15/typescript/utils/src/addresses.ts#L220